### PR TITLE
Adding two slovak books about Linux

### PR DESCRIPTION
**Pull request by fusaty**
## What does this PR do?
Add resources

### Description
Adding two free slovak books about Linux
### Why is this valuable (or not)?
There is very little books in slovak in this repository. Want to help a little with that.
### How do we know it's really free?
Greenie knižnica, which holds this book is a slovak portal for free books.
### For book lists, is it a book? For course lists, is it a course? etc.
Its two books.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by fusaty** [comment]
Hi, you added "waiting for changes" label. What changes are needed?

**Comment by eshellman** [comment]
use "Operating Systems", not "Linux"

**Comment by fusaty** [comment]
> use "Operating Systems", not "Linux"

fixed

File: books/free-programming-books-sk.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 * [Language Agnostic](#language-agnostic)
     * [Právo](#pravo)
+    * [Všeobecné programovanie](#vseobecne-programovanie)
+* [Operačné systémy](#operacne-systemy)
+    * [Linux](#linux)
```
Comment (eshellman) [inline books/free-programming-books-sk.md:7]: just put the links in the Operačné systémy category, don't need a subcategory when there are only two resources

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-sk.md (hunk 1 above)
Comment (fusaty) [reply books/free-programming-books-sk.md]: done

**Pull request merged**